ui/src/components/Form.jsx

```diff
@@ -39,6 +42,18 @@ function Form() {
     })();
   }, [displayError]);
 
+  function toggleAttribute(attribute) {
+    const attributeIndex = selectedAttributes.findIndex(attr => attr.id === attribute.id);
+    if (attributeIndex !== -1) {
+      setSelectedAttributes((attributes) =>
+        attributes.filter((item, index) => index !== attributeIndex)
+      );
+    } else {
+      setSelectedAttributes((attributes) => [...attributes, attribute]);
+    }
+    console.log(selectedAttributes);
```

Comment:
_nitpick_: not important right now, but keep in mind to remove stuff like this in the future projects 😄

Reply:
Oops. Good catch.